hoomd/mpcd/CollisionMethod.cc

```diff
@@ -48,15 +78,945 @@ void mpcd::CollisionMethod::collide(uint64_t timestep)
     // sync the embedded group
     m_cl->setEmbeddedGroup(m_embed_group);
 
+    // check for collision warnings the first time collide is run
+    if (!m_checked_collision_warnings)
+        {
+        checkCollisionWarnings(timestep);
+        m_checked_collision_warnings = true;
+        }
```

Comment:
I prefer to space out if statements to make sure it's clear these aren't an if-else if
```suggestion
        }

```